apps/wallet/src/features/Home/index.tsx

```diff
@@ -2,27 +2,51 @@ import type { ViewStyle } from 'react-native';
 import { StyleSheet } from 'react-native';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 import { Networks } from '@walless/core';
-import { View } from '@walless/gui';
+import { Button, View } from '@walless/gui';
+import { Setting } from '@walless/icons';
 import FeatureButtons from 'components/FeatureButtons';
 import { showReceiveModal } from 'modals/Receive';
 import { showSendTokenModal } from 'modals/SendToken';
+import { appState } from 'state/app';
 import { buyToken } from 'utils/buy';
 import { tabBarHeight } from 'utils/constants';
-import { useTokens } from 'utils/hooks';
+import { useSnapshot, useTokens } from 'utils/hooks';
+import { navigate } from 'utils/navigation';
 
 import History from './History';
 import TokenValue from './TokenValue';
 
 const ProfileFeature = () => {
 	const insets = useSafeAreaInsets();
+	const { isMobileDisplay } = useSnapshot(appState);
 	const { valuation } = useTokens();
 	const containerStyle: ViewStyle = {
 		paddingBottom: tabBarHeight + insets.bottom,
 	};
+	const handleSettingPress = () => {
+		navigate('Dashboard', {
+			screen: 'Explore',
+			params: {
+				screen: 'Profile',
+				params: {
+					screen: 'Setting',
+				},
+			},
+		});
+	};
```

Comment:
wonder if we can make this method reuseable? seems that it could be a static constant (no hook or effect values being used, and the setting screen also well defined without component level context)

Reply:
this setting btn only shows up in extension thus this route is not available in mobile layout. Therefore I find it's not usable anywhere else.